packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -124,11 +129,15 @@ export class ChatWidget extends Component {
       // If new activity comes in
       if (lastActivityFromPrev && lastActivityFromThis && props.conversation.activities.length !== prevProps.conversation.activities.length && lastActivityFromPrev.id !== lastActivityFromThis.id) {
         // Scroll if from ourselves
-        if (props.user.currentUser.id === lastActivityFromThis.actorId) {
+        if (props.user.currentUser.id === lastActivityFromThis.actor.id) {
```

Comment:
Good catch. Looks like a remanent from when we weren't grabbing the full activity object.